packages/vscode-graphql-execution/src/providers/exec-content.ts

```diff
@@ -37,10 +37,8 @@ export class GraphQLContentProvider implements TextDocumentContentProvider {
 
   timeout = (ms: number) => new Promise(res => setTimeout(res, ms));
 
-  getCurrentHtml(): Promise<string> {
-    return new Promise(resolve => {
-      resolve(this.html);
-    });
+  async getCurrentHtml(): Promise<string> {
```

Comment:
I think this should be sync too
```suggestion
  async getCurrentHtml(): string {
```